README.md

```diff
@@ -26,21 +26,22 @@ Make sure you have these environment variables set:
 ```
 DATABASE_URL=postgres://USERNAME:PASSWORD@HOST:PORT/DBNAME
 SECRET_KEY='abcd1234'
+LAND_AVAILABILITY_API_URL=http://localhost:8000
```

Comment:
Sure, I'm just trying to standardize defaults. I'm starting from what you default to in import_cambridge_lands.py:
* 8000 api
* 8001 lr
* 8002 voa